main/modes/platformer/entityManager.c

```diff
@@ -63,6 +63,9 @@ void loadSprites(entityManager_t * entityManager)
     loadWsg("sprite021.wsg", &entityManager->sprites[SP_WARP_1]);
     loadWsg("sprite022.wsg", &entityManager->sprites[SP_WARP_2]);
     loadWsg("sprite023.wsg", &entityManager->sprites[SP_WARP_3]);
+    loadWsg("sprite024.wsg", &entityManager->sprites[SP_WASP_1]);
```

Comment:
I'm not gonna require a change because you've got so many sprites, but next time try to use more descriptive and compact names. SPIFFS doesn't have folders for the output, so there's a chance of colliding with another mode's generically named sprites. I'd have done something like `ps_wasp1.wsg` for this line (`ps` for 'platformer sprite').

Reply:
They are named like this so that I can export all my sprites from my pixel art editor with 1 command, and also to keep the names consistent with where they are loaded into the sprites array.

Reply:
Gotcha